### PR TITLE
PIC-3156: Unlink nDelius page button/cancel link alignment

### PR DESCRIPTION
**Pull request by KiraNamida**
PIC-3156: Create a component for the GDS standard button input and cancel link along with int test steps.

image

### CODE REVIEW

File: server/views/components/button-group/template.njk [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<div class="govuk-button-group button-group">
+    <button class="govuk-button action-button">{{buttonText}}</button>
```
Comment (Obsiye) [inline server/views/components/button-group/template.njk:2]: i don't know this service well, sorry.

Is it ok that the button is defined a little differently [here](https://github.com/ministryofjustice/prepare-a-case/blob/main/server/views/components/button/template.njk) ? or should it use button-group too?

` <button id={{id}} type={{type}} class="action-button {{classes | join(' ')}}">{{text}}</button>`

Comment (KiraNamida) [reply server/views/components/button-group/template.njk:2]: I would definitely like an institutional opinion on that. I went with the "minimum change required" approach in the first pass. Perhaps I should look at using that button inside this template to we still have one definition of a button component. 🤔

Comment (KiraNamida) [reply server/views/components/button-group/template.njk:2]: Although, I have just dug for other references of that macro and it is only being imported in one file where it is not referenced and if I do use it, it throws a render error so something is going on here... 🙃

Comment (Obsiye) [reply server/views/components/button-group/template.njk:2]: no worries, happy for you to ignore this comment 👍🏾

**Review by johnyfarrar — APPROVED** [review]
Yeah I like it! LGTM

**Pull request merged**